packages/plugins/foundry-contracts/ChugSplash.sol

```diff
@@ -58,17 +73,76 @@ contract ChugSplash is Script, Test {
         }
     }
 
-    function _initializeChugSplash() private {
-        string[] memory cmds = new string[](7);
+    function getBootloaderBytecode() private returns (DeploymentBytecode memory) {
+        string[] memory cmds = new string[](4);
         cmds[0] = "npx";
         cmds[1] = "node";
         cmds[2] = filePath;
-        cmds[3] = "initializeChugSplash";
-        cmds[4] = rpcUrl;
-        cmds[5] = network;
-        cmds[6] = privateKey;
+        cmds[3] = "getBootloaderBytecode";
+
+        bytes memory result = vm.ffi(cmds);
+        return abi.decode(result, (DeploymentBytecode));
```

Comment:
The bootloaders are not deployed here. The purpose of this function is to fetch the exact bytecode of the bootloaders from the artifacts on the contracts package which are then deployed using create 2. `type(bootloader).creationCode` doesn't work because the bytecode generated is dependent on the users' compiler configuration which may be different from ours.

Reply:
Yeah, that's fine, I'll update this to use a separate getter here instead of including the address on the same struct.